pom.xml

```diff
@@ -83,7 +83,13 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>github-branch-source</artifactId>
-            <version>2.3.6</version>
+            <version>2.5.4-rc860.9c636841daa2</version>
```

Comment:
is this correct, seems like a release candidate..?

otherwise thanks for the PR!

Reply:
I didn't mention that particular version should be replaced as soon as https://github.com/jenkinsci/github-branch-source-plugin/pull/235 is merged and released. Sorry about the lack of communication

Reply:
Can be set to 2.6.0 now that https://github.com/jenkinsci/github-branch-source-plugin/pull/235/commits/bb2782d966b263bff08db28b7dd0c0dfadb837db has been released.